File: greensopinion.finance.services/src/greensopinion/finance/services/AboutService.java
package greensopinion.finance.services;

import greensopinion.finance.services.model.About;

public class AboutService {

	public About getAbout() {
		return new About(getApplicationName(), getCopyrightNotice());
	}

	private String getApplicationName() {
		return "Green's Opinion - Finances";
	}

	private String getCopyrightNotice() {
		return "Copyright (c) 2015 David Green.  All rights reserved.";
	}
}

File: greensopinion.finance.ui/app/scripts/services/aboutService.js
/*******************************************************************************
 * Copyright (c) 2015 David Green.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the EULA
 * which accompanies this distribution.
 *******************************************************************************/
'use strict';

angular.module('greensopinionfinanceApp')
  .service('aboutService',['rest',function(rest) {
    var API_BASE = '/abouts/';
    return {
      about: function() {
          return rest.get(API_BASE+'current');
      }
    };
  }]);
